docs/updates/2019-06.md

```diff
@@ -0,0 +1,121 @@
+# June 2019
+
+Welcome to the June 2019 Qlik Core update. There are a number of updates in this version that we hope you will like,
+some of the key highlights include:
+
+## New features in corectl
+
+Since the launch of corectl we have received a couple of feature request from our users which we love to implement.
+Thank you Jesse Paris for all your feedback!
+
+The new version [1.2.0](https://github.com/qlik-oss/corectl/releases/tag/v1.2.0) includes all the latest features.
+As always, you can get the latest by using `brew upgrade`, `snap refresh` or downloading from
+[GitHub](https://github.com/qlik-oss/corectl/releases).
+
+### Connect to Qlik Sense Enterprise for Windows with certificates
+
+If you would like to connect to Qlik Sense Enterprise for Windows using certificates (instead of JWT validation) we have
+added support through the new certificates parameter. To enable this, create new certificates in your Qlik Sense
+Environment. Export them to a location where corectl can reference them and add the `--certificates` to
+your corectl parameters.
+
+As an example, you can list a user's application by:
+
+```bash
+corectl --engine wss://<qlik-sense-url>:4747 --certificates <certfolder> --headers "X-Qlik-User: UserDirectory=<userdir>; UserId=<userid>" app ls
+```
+
+```txt
++--------------------------------------+------+--------------------------+----------+-------+
+|                  Id                  | Name |      Last-Reloaded       | ReadOnly | Title |
++--------------------------------------+------+--------------------------+----------+-------+
+| ea39af16-e6bb-4aa5-bea3-3386b6ef307b | test | 2019-06-26T08:30:34.155Z | false    | test  |
++--------------------------------------+------+--------------------------+----------+-------+
+```
+
+To keep the command shorter, the configuration can be placed in a corectl.yml file. You can read more about the feature
+in the [corectl config](https://github.com/qlik-oss/corectl/blob/master/docs/corectl_config.md#certificates) chapter.
+
+### Unbuild - export app content
+
+![screenshot](../images/corectl-unbuild.gif)
```

Comment:
Great idea with an animated gif! I think we should show something else than the connections file though. Probably more interesting to show e.g. that all the objects were exported.